plugins/app-backend/migrations/20211229105307_init.js

```diff
@@ -25,12 +25,11 @@ exports.up = async function up(knex) {
     table.comment(
       'A cache of static assets that where previously deployed and may still be lazy-loaded by clients',
     );
+    table
+      .increments('id', { primaryKey: true })
+      .comment('Primary key to distinguish unique lines from each other');
     if (!isMySQL) {
-      table
-        .text('path')
-        .primary()
-        .notNullable()
-        .comment('The path of the file');
+      table.text('path').unique().notNullable().comment('The path of the file');
```

Comment:
Doesn't https://github.com/backstage/backstage/pull/20409/files#diff-758e9368aaa34d6e9f5f81ea57bbebe3064db6a0d675af9539570039d91ec346R47-R51 add a partial uniqueness constraint for `path`?

That said, I'm worried that constraint will cause false collisions in MySQL, since the beginning of a path is most likely to collide.